src/pages/restaurant/components/dishItems/components/menu/Menu.js

```diff
@@ -4,25 +4,28 @@ import PropTypes from "prop-types";
 // lodash
 import _map from "lodash/map";
 import _startCase from "lodash/startCase";
+import _noop from "lodash/noop";
 
 // components
 import MenuSection from "./components/menuSection";
 
 // css
 import "./menu.css";
 
-const renderMenuSection = (itemValuesList, itemKey) => {
-  return (
-    <MenuSection
-      key={itemKey}
-      itemList={itemValuesList}
-      heading={_startCase(itemKey)}
-    />
-  );
-};
-
 const Menu = function (props) {
-  const { items, className } = props;
+  const { items, className, cart, onCartChange } = props;
+
+  const renderMenuSection = (itemValuesList, itemKey) => {
```

Comment:
inline function

Reply:
In my opinion, this would be a better approach